### PR TITLE
Add about page

### PR DESCRIPTION
**Pull request by Raytray**
* chmod -x images
* Made some minor styling fixes... There were unclosed divs
* remove unused files for about

screen shot 2018-11-12 at 13 37 46
screen shot 2018-11-12 at 13 37 57

### CODE REVIEW

File: layouts/partials/nav.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <img src="/img/logo@3x.png" height="40" style="margin: 5px" />
+            <a href="/"><img src="/img/logo@3x.png" height="40" style="margin: 5px" /></a>
```
Comment (Raytray) [inline layouts/partials/nav.html:12]: Made it relative link to home page, not sure if this was left out on purpose

File: config.toml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-baseurl = "https://bluestonehomeloans.netlify.com/"
```
Comment (Raytray) [inline config.toml:1]: Don't know if this should have been changed, but it seemed like it shoudl have..

File: layouts/partials/why.html [hunk 3]
```diff
@@ -41,5 +41,6 @@
                     </div>
             </div>
         </div>
+    </div>
```
Comment (Raytray) [inline layouts/partials/why.html:44]: I added a closing div, thought the partial should close itself, but if that's not true let me know

**Review by Raytray — COMMENTED** [review]
@floofydoug thoughts on the specific comments?

File: layouts/partials/why.html (hunk 3 above)
Comment (floofydoug) [reply layouts/partials/why.html:44]: oops i think i left that out. Thanks for the catch. When i merge this, it should go live.

**Comment by floofydoug** [comment]
Cool! Really appreciate it, Ray. What do you think of Hugo so far?

**Comment by floofydoug** [comment]
hrm, after merging, the title on About is pretty massive. Might try to make it smaller....

**Comment by Raytray** [comment]
I used the hero partial on the home. I thought the same but left it not
wanting to affect the index.

The problem is that the creative submodule defines the header to be
min-width 100%

Hugo is cool, a lot of the same principals of jekyll and the like so it
made it straight forward to pick up.

edit: I'll take a look at overriding the css to make it more reasonable though

**Pull request merged**